matrix.yml

```
@@ -1,7 +1,7 @@
 ansible-core:
   - 2.12.10
-  - 2.13.6
-  - 2.14.0
+  - 2.13.7rc1
+  - 2.14.1rc1
```

We keep the previous GA releases until the next one so during the RC releases we're testing N-1 GA and N RC at the same time.

```suggestion
  - 2.13.6
  - 2.13.7rc1
  - 2.14.0
  - 2.14.1rc1
```

---

Gotcha, let me adress